src/Commands/ListAddListCommand.ts

```diff
@@ -0,0 +1,32 @@
+import { NativeCallProperties } from "../Languages/Properties/NativeCallProperties";
+import { NativeCallCommand } from "./NativeCallCommand";
+import { Parameter } from "./Parameters/Parameter";
+import { SingleParameter } from "./Parameters/SingleParameter";
+
+/**
+ * A command that adds everything in the second list to the first list.
+ */
+export class ListAddListCommand extends NativeCallCommand {
+    /**
+     * Information on parameters this command takes in.
+     * second parameter contains values that will be added to first parameter
+     */
+    private static parameters: Parameter[] = [
+        new SingleParameter("name", "The name of the list.", true),
+        new SingleParameter("name", "The name of the list.", true)
```

Comment:
By descriptions I meant what's in the strings. `"name"` and `"the name of the list."` aren't very descriptive. How about `"first"` and `"second"`?